roles/install_yamls/README.md

```diff
@@ -57,3 +57,35 @@ The role can be imported and tasks can be executed like this
 
 Tasks are created in a form of: `make_{{ original make target }}`, corresponding to `make target` imported from [a Makefile](https://github.com/openstack-k8s-operators/install_yamls/blob/c8487df41bf9ddefa7989f9384e77ae9720ce9dd/Makefile#L418).
 When task is being executed, it runs corresponding code from the Makefile.
+
+## Make install_yaml Parameter
+
+### module: generate_make_tasks
+
+1. For every makefile target we have a dedicated task file.
+
+```YAML
+    options:
+      install_yamls_path:
+      description:
+        - Absolute path to install_yamls repository.
+      required: True
+      type: str
+      output_directory:
+      description:
+        - Absolute path to the output directory. It must exists.
+      required: True
+      type: str
+```
+
+2. Follow below steps to find makefile target dedicated task files
+
+Connect to the controller-0 node and go to ci-framework directory
+```Bash
+  [laptop]$ cd ~/src/github.com/openstack-k8s-operators/ci-framework
+  [laptop]$ cd ci-framework-data/artifacts/roles/install_yamls_makes/tasks
+```
+
+### Overide install_yaml make parameter
+
+You can build specific parameters using make_TARGET_env and make_TARGET_dryrun".
```

Comment:
I don't get this point, neither the terms used here:

- `make_TARGET_env`: That passes an environment variable, that is **not** a make parameter. The way we generate the task using the generate_make_task module doesn't allow us to use parameters.
- - `make_TARGET_dryrun`: Not a param, it's an Ansible variable that doesn't go outside Ansible, in other words, make has no idea about its existence.